core-api/src/main/java/com/sequenceiq/cloudbreak/doc/OperationDescriptions.java

```diff
@@ -59,6 +59,8 @@
         public static final String DATABASE_RESTORE_INTERNAL = "Performs a restore of the database from a provided location, internal only";
         public static final String ROTATE_CERTIFICATES = "Rotates the certificates of the cluster";
         public static final String RENEW_CERTIFICATE = "Trigger a certificate renewal on the desired cluster which is identified via stack's name";
+        public static final String UPDATE_LOAD_BALANCERS = "Updates an existing cluster with load balancers, including adding the Endpoint Gateway " +
+            "if it's enabled.";
```

Comment:
I think this line can be removed.

Reply:
It's used here: https://github.com/hortonworks/cloudbreak/pull/9940/files#diff-25a6dcfa1c517acc3b181311af4acdadb603914209d2ea7414441cdea1dee574R438